util/config/schema.py

```diff
@@ -1536,5 +1536,22 @@
             "description": "Allows organization administrators to assign OAuth tokens to other users",
             "x-example": False,
         },
+        "DEFAULT_NAMESPACE_AUTOPRUNE_POLICY": {
+            "type": "object",
+            "description": "Default auto-prune policy applied to newly created organizations",
+            "properties": {
+                "method": {
+                    "type": "string",
+                    "description": "",
+                    "enum": ["number_of_tags", "creation_date"],
+                    "x-example": "number_of_tags",
+                },
+                "value": {
+                    "type": "string",
```

Comment:
config tool checks that if method type is `number_of_tags`, value must be `int`. And I see that is what we check in [validation](https://github.com/quay/quay/blob/master/data/model/autoprune.py#L80) too

Reply:
good point, updated